src/interfacemain.cpp

```diff
@@ -41,6 +45,21 @@ int main( int argc, char *argv[] )
 	qDebug() << "Usage: ./oneclickhelper <File Path>";
 	return 1;
     }
+ 
+    // Progress callbacks
+    try {
+	static RpmCallbacks rpmCallbacks;
+	static SourceCallbacks sourceCallbacks;
+	static MediaCallbacks mediaCallbacks;
+    }
+    catch ( const Exception& e )  {
+	qDebug() << "Failed to initialize OCI callbacks.";
```

Comment:
If we're going to exit without telling anything to the user, let's at least make it a qFatal so it's printed on stderr